examples/pom.xml

```diff
@@ -203,23 +209,35 @@
         </exclusion>
         <exclusion>
           <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-common</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey</groupId>
           <artifactId>jersey-core</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>org.slf4j</groupId>
-          <artifactId>slf4j-api</artifactId>
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-json</artifactId>
         </exclusion>
         <exclusion>
           <groupId>com.sun.jersey</groupId>
           <artifactId>jersey-server</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>com.sun.jersey</groupId>
-          <artifactId>jersey-core</artifactId>
+          <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
The issue is that some third-party deps may still be using the old Jersey. Right? I think Hadoop might. If it's `javax` problems, yeah I think we need to tackle that one directly. You might see if it all just works without exclusions, and then investigate the dependency tree, and sense-check whether the various versions that end up being packaged make sense.